viral/imaging_utils.py

```
@@ -365,13 +371,19 @@ def get_ITI_matrix(
     matrices = []
 
     for trial in trials:
-        assert trial.trial_end_closest_frame is not None
+        end_ITI = (
+            trial.trial_end_closest_frame
+            if trial.trial_end_closest_frame is not None
+            else [state for state in trial.states_info if state.name == "ITI"][
+                0
+            ].closest_frame_end
```

Ruff (0.13.2)

Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)